index.html

```diff
@@ -26,49 +26,289 @@
     <title>Document</title>
   </head>
   <body>
-    <header>
-      <nav>
-        <a href="#" class="myLogo"> <span>My Logo</span></a>
-        <i class="fa-solid fa-bars fa-xl myBar"></i>
-      </nav>
-    </header>
-    <section>
-      <div class="headline">
-        <h1>
-          I’m Oscar <br />
-          Glad to see <br />
-          you!
-        </h1>
+    <div class="contaner">
+      <header>
+        <nav>
+          <a href="#" class="myLogo"> <span>My Logo</span></a>
+          <i class="fa-solid fa-bars fa-xl myBar"></i>
+        </nav>
+      </header>
+      <section class="section-1">
+        <div class="headline">
+          <h1>
+            I’m Oscar <br />
+            Glad to see <br />
+            you!
+          </h1>
+        </div>
+        <div class="about-me">
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+        </div>
+        <a class="meet">LET'S CONNECT</a>
+        <div class="social-icons">
+          <ul class="social-icons-list">
+            <li>
+              <a href="#"><img src="assests/images/Vector.png" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="assests/images/Vector (1).png" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="assests/images/Vector (2).png" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="assests/images/Vector (3).png" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="/assests/images/Linkedin icon.png" alt=""
+              /></a>
```

Comment:
Kindly, I suggest that for accessibility every time you use an "IMG" you indicate an appropriate "ALT" to describe the content of that image